include/parallel/standard_type.h

```diff
@@ -27,6 +27,7 @@
 
 // C++ includes
 #include <numeric>
+#include <tuple>
```

Comment:
Actually, what's annoying here is that this PR will *not* be the first use in the library, because that specialization doesn't yet get used.  I misread one of the chunks of code I intended to refactor, I concluded *erroneously* that I'd need tuple communication support for it right away, and I wrote up the Parallel tuple support before realizing my mistake.

If you'd like I can remove the tuple commit and save it for a PR where it actually gets exercised.